contrib/debian/control.jessie

```diff
@@ -40,8 +40,7 @@ Depends: adduser,
          libjudydebian1,
          openssl,
          libmnl0,
-         libjson-c2,
-         freeipmi,
+         libjson-c3,
```

Comment:
@paulkatsoulakis 

Sorry to comment into a closed PR, but I can't install release v1.20 on Debian Jessie because libjson-c3 is not available (only libjson-c2). This was "wrong" already once and I made PR to change it (#7342). Is there any reason why this change was reverted with this PR?

Reply:
This is almost certainly my fault, I wasn't enough attention while fixing the merge conflicts. I'll have a PR up to fix this again shortly after which I'll look into re-publishing the packages for 1.20 for Jessie so they have the fix.

Reply:
#8290